features/autorisatie/bewoning/raadpleeg-bewoning-met-periode/autorisatie.feature

```diff
@@ -28,7 +28,7 @@ Functionaliteit: autorisatie voor het gebruik van de API BewoningMetPeriode
       | 0518                 | 0518010000000002                         |
 
 
-  Regel: Een gemeente als afnemer is geautoriseerd voor het vragen van bewoning waarbij voor de gevonden verblijfplaats(en) de gemeente van inschrijving gelijk is aan de gemeentecode in de 'claim' uit de token
+  Regel: Een gemeente als afnemer is geautoriseerd voor het vragen van bewoning waarbij de gemeente van de gevonden verblijfplaats(en) gelijk is aan de gemeentecode van de afnemer.
```

Comment:
Dit is volgens mij niet wat wij afgesproken hebben. In dit geval moeten de verblijfplaatsen zijn opgehaald om dit te bepalen. We zouden het nu bepalen adhv de gemeentecode van de adres voorkomens van het adresseerbaar object. Volgens mij zou deze regel nu moeten zijn:

Een gemeente als afnemer is geautoriseerd voor het vragen van de bewoning van een adresseerbaar object als het adresseerbaar object ooit in de gemeente van de afnemer heeft gelegen